eoscostarica/eosio-dashboard
# feat: add tps all time high

### feat: add tps all time high

### What does this PR do?

- Resolve #509

### Steps to test

1. make run laccahin
2. navigate to http://localhost:3000/
3. check the new card "TPS All Time High"

#### CheckList
